cl/citations/tasks.py

```diff
@@ -125,20 +124,29 @@ def find_citations_and_parentheticals_for_opinion_by_pks(
     )
     for opinion in opinions:
         try:
-            store_opinion_citations_and_update_parentheticals(opinion)
+            # delivery_info does not exist in test environment
```

Comment:
I must be missing something, but why not just pass in the queue from the jump so you dont need to call it here.  and pass it along to `store_opinion_citations_and_update_parentheticals`

Reply:
fair enough.